src/Core/Entities/Provider/ProviderOrganization.cs

```diff
@@ -1,14 +1,16 @@
 ﻿using Bit.Core.Utilities;
 
+#nullable enable
+
 namespace Bit.Core.Entities.Provider;
 
 public class ProviderOrganization : ITableObject<Guid>
 {
     public Guid Id { get; set; }
     public Guid ProviderId { get; set; }
     public Guid OrganizationId { get; set; }
```

Comment:
this is nullable in DB, but if code is not nullable, please follow code.

Reply:
It is and the discrepancy should be addressed but that would be a real code change since that is a null value type vs me trying to annotate the null reference types in this PR.